File: python/conversao_monetaria.py
print("conversor de moedas")
escolha = input("primeiro deseja converter dolar ou real?").lower()

if escolha.isalpha() == False:
  print("nao vai não em")

elif escolha.isalpha() == True:
  if escolha == "dolar":
    dolar = int(input("digite o valor para converter: "))
    conversão = dolar * 5
    print(f"{dolar} dolar/es convertido/s para real da {conversão} real/ais")
    
  elif escolha == "real":
    real = int(input("digite o valor para converter: "))
    conversão = real / 5
    print(f"{real} real/ais convertido/s para dolar da {conversão} dolar/es")

File: javascript/conversao_monetaria.js
console.log("Conversor de moedas");
let escolha = prompt("Primeiro, deseja converter dólar ou real?").toLowerCase();

if (!isNaN(escolha)) {
  console.log("Não vai não em");
} else {
  if (escolha === "dolar") {
    let dolar = parseInt(prompt("Digite o valor para converter: "));
    let conversao = dolar * 5;
    console.log(`${dolar} dólar(es) convertido(s) para real dá ${conversao} real(ais)`);
  } else if (escolha === "real") {
    let real = parseInt(prompt("Digite o valor para converter: "));
    let conversao = real / 5;
    console.log(`${real} real(ais) convertido(s) para dólar dá ${conversao} dólar(es)`);
  }
}